packages/focal/src/react/react.ts

```diff
@@ -76,50 +93,45 @@ export class LiftWrapper<TProps>
       // Could this be replaced by a regular closure? Perhaps using
       // a class is an optimization?
       case 1:
-        new RenderOne(this, newProps) // eslint-disable-line
+        new RenderOne(renderCache, newProps, setCache, setSubscription) // eslint-disable-line
         break
       default:
-        new RenderMany(this, newProps, n) // eslint-disable-line
+        new RenderMany(renderCache, newProps, setCache, setSubscription, n) // eslint-disable-line
         break
     }
-  }
 
-  private _unsubscribe() {
-    const subscription = this.state ? this.state.subscription : null
-    if (subscription)
-      subscription.unsubscribe()
+    inSync = true
```

Comment:
What is the intent of setting it into `true`? Does that mean, that we turning off `forceUpdate` for the initial render, but calling it for each new emit? If so, does it make sense to call `forceUpdate` here to force render of the latest cache?

Reply:
Added comment and rename variable to `isSubscribed`